framework/Util/IBaseBehavior.php

```diff
@@ -22,6 +22,16 @@
  */
 interface IBaseBehavior
 {
+	/**
+	 * The array key for the $config data in instancing behaviors with init($config),
+	 */
+	public const CONFIG_KEY = "=config";
```

Comment:
Is "=config" intended to start with "="? To avoid people from setting it accidentally?

Reply:
exactly.

Reply:
'=' isn't a supported character for tags in xml.